src/MerchantCenter/MerchantStatuses.php

```diff
@@ -168,13 +160,16 @@ public function get_product_statistics( bool $force_refresh = false ): array {
 	 * @throws Exception If the account state can't be retrieved from Google.
 	 */
 	public function get_issues( string $type = null, int $per_page = 0, int $page = 1, bool $force_refresh = false ): array {
```

Comment:
💅  What do you think about updating the return docs?

`* @return array With two indices, results (may be paged) and count (considers type).`

Now we are returning 'loading' as well